pilot/pkg/config/kube/gateway/testdata/eastwest.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: Gateway
+metadata:
+  name: eastwestgateway
+  namespace: istio-system
+  labels:
+    topology.istio.io/network: "network-1"
+spec:
+  gatewayClassName: istio
+  listeners:
+  - name: istiod-grpc
+    port: 15012
+    protocol: TLS
+    tls:
+      mode: Passthrough
+  - name: istiod-webhook
+    port: 15017
+    protocol: TLS
+    tls:
+      mode: Passthrough
+  - name: cross-network
```

Comment:
Would we break if someone specified multiple listeners with the same port but different hosts? e.g. `*.local` and `serviceentry.com`

Reply:
Good point. In today code they can set a list of hostnames. I will see what happens here